docs/configuration.markdown

```diff
@@ -194,6 +194,10 @@ end
   "cirrus", "vmvga", "xen", "vbox", or "qxl".
 * `video_vram` - Used by some graphics card types to vary the amount of RAM
   dedicated to video.  Defaults to 16384.
+* `video_resolution_x` - sets the default EDID/monnitor resolution hinting for the X
```

Comment:
While spitting the resolution into two separate variables makes it easier in some ways, I'm not sure it's particularly consistent with how many would expect it to work with `1280x800` being a more common form. Should be possible to validate the value matches `[[:digit:]]{3,}x`[[:digit:]]{3,}` (or similar regex) if not nil in the config class and subsequently split when specifying the individual XML elememts